AdminServer/appscale/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
The setuptools docs claim that line is required for namespace packages: http://setuptools.readthedocs.io/en/latest/setuptools.html#namespace-packages

`import` probably works too, but I think it would cause `appscale.pkg_resources` to be defined.